frontend/app/routes/application.js

```diff
@@ -77,7 +77,7 @@ export default Ember.Route.extend(ApplicationRouteMixin, {
 
     var flowSelectionArray = Ember.A([]);
     model.flows.forEach(function(flow){
-     flowSelectionArray.push({label: flow.name, value: flow.id});
+     flowSelectionArray.push({label: flow.name, value: flow.id, selectable: flow.selectable});
```

Comment:
Should this push only selectable flows instead? Is the flowSelectionArray ever used without filter on selectable? If not, this would be cleaner to filter here at the source instead of in the templates.